common/src/reserved.rs

```diff
@@ -507,6 +513,43 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_apply_delegate_on_consensus_failure() {
+        // given
+        setup_test();
+        let (mut state, keys) = generate_standard_genesis(3);
+
+        // delegator: member-0000, delegatee: member-0002
```

Comment:
How about deleting this line? I think copy and pasting this line is not necessary and misleading.